waiter/specs/k8s-default-pod.edn

```diff
@@ -0,0 +1,41 @@
+{:kind "ReplicaSet"
+ :apiVersion "extensions/v1beta1"
+ :metadata {:name #waiter/param :app-name
+            :labels {:app #waiter/param :app-name
+                     :managed-by "waiter"}
```

Comment:
`:managed-by` is something I made up. It would be good if we can find an example of a more idiomatic label from the k8s community that has a similar connotation.